src/components/EditCoursePage/index.jsx

```diff
@@ -2,12 +2,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Helmet from 'react-helmet';
-import moment from 'moment';
 
 import EditCourseForm from './EditCourseForm';
 import StatusAlert from '../StatusAlert';
 import LoadingSpinner from '../LoadingSpinner';
-import { getCourseNumber, jsonDeepCopy } from '../../utils';
```

Comment:
I think we still need the deep copy on the course runs since https://github.com/edx/publisher-frontend/blob/d57ff232fd11713e8ef35e1e3093d8b0f92c017c/src/components/EditCoursePage/index.jsx#L36-L40 directly modifies the staff

Reply:
Ahh gotcha, I actually messed up and rebased out the staff work originally so thanks for the catch here